### ISSUE TITLE
Allow API token to be scope to multiple projects

### ISSUE THREAD

**Issue by ashb** [pypi/warehouse#12176]
Hello, I'm a member of the core team for Apache Airflow, and we have our project split up in to a number of distributions on PyPI - the core "apache-airflow" dist and (currently, but it grows over time) 70 provider modules, apache-airflow-providers-*.

I would like to be able to "easily" upload those provider dists via API tokens, but currently I either need to use an account-wide token, or each upload needs a separate API key (which doesn't easily work for our normal release workflow, where we will release all provider packages with changes once a month)

It's possible Organizations might help here if it's possible to have an Org-wide API token?

Relates to #7265

**Comment by dstufft** [comment]
You can create an account wide token, then scope it yourself for multiple projects if that works for you, although I do agree ti would be nice for our UI to support multiple of them here.

**Comment by di** [comment]
Hi, thanks for the issue. We agree this is a good idea, and that it's going to be much easier to implement to implement for organizations/teams once that feature has landed, so we'll consider this blocked on https://github.com/pypi/warehouse/issues/11590.

**Comment by Julian** [comment]
Is this a duplicate of #6292 perhaps?

**Comment by di** [comment]
Yep! Duplicate of #6292.

**Comment by ttodua** [comment]
> You can create an account wide token, then scope it yourself for multiple projects if that works for you, although I do agree ti would be nice for our UI to support multiple of them here.

Hi @dstufft , can you tell how to scope it to specific projects (after creating it for entire account scope)? can't find any way in UI.
